### DISCUSSION TITLE
添加一个开机启动并且最小化的功能

### DISCUSSION THREAD

**Discussion by mqq2068** [Planshit/Tai#212] [Ideas]
之前开机启动有点bug，但是多尝试了几次之后现在可以实现开机启动，但是每次开机都在最前面，还要关掉。
![image](https://github.com/Planshit/Tai/assets/67449016/77cb1695-14ac-44ea-9c16-f0703f254976)
类似于这样的

**Comment by noberumotto** [comment]
下一次更新会去掉开机自启时显示主界面
